src/Transposit.ts

```diff
@@ -227,3 +237,26 @@ export interface SignInSuccess {
 export interface OperationParameters {
   [paramName: string]: string;
 }
+
+class EndRequestLogResponse<T> implements OperationResponse<T> {
+  results: T[];
+  requestId: string;
+  constructor(log: EndRequestLog, response: Response) {
+    this.requestId = log.requestId;
+    const logResults = log.result.results;
+    if (!logResults) {
+      throw new APIError(
+        "API returned an unexpected response schema",
+        response,
+      );
+    }
+    this.results = logResults;
+  }
+
+  get value(): T {
+    if (this.results.length === 0) {
```

Comment:
Should we assert this is used when the value is exactly 1?

Reply:
I have no strong opinion. We had discussed it and said it should always just have the first entry, but at that point you may have assumed we'd use a normal object; if you feel strongly that it should only get used when we only return one result, I feel okay with that.

Alternatively, if you mean that we assert they _don't_ use the results array when they only have one entry, we definitely shouldn't because operations may return 1 result some but not all of the time.